### ISSUE TITLE
Filename of saved media?

### ISSUE THREAD

**Issue by lucagoetheil** [wireapp/wire-android#2959]
I'm trying to download all images in a conversation. It looks like I have to do this manually, but I'm wondering if there's a way to associate the timestamp that the image was sent/received to the filename of the image. 

How are filenames currently decided for the images? Is there any way to change this?

**Comment by lucagoetheil** [comment]
Could anyone help me out with this? Thanks.

**Comment by kroerig** [comment]
Would be even nice to choose where to save the downloaded images.

**Comment by mosiser** [comment]
Indeed, pictures downloaded are without metadata and exif, so it's nearly impossible to order them by time. A filename format including sent or received timestamp would be the best.
